apps/webClient/src/views/LetsTalk/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { makeStyles } from "@material-ui/styles";
+import Grid from "@material-ui/core/Grid";
+import Box from "@material-ui/core/Box";
+import FlareIcon from "@material-ui/icons/Flare";
+import Typography from "@material-ui/core/Typography";
+import { Theme } from "@material-ui/core/styles/createMuiTheme";
+import Logo from "../../components/Logo/";
+import NavButton from "../../components/NavButton";
+
+interface Props {}
+
+const useStyles = makeStyles((theme: Theme) => ({
+  contrastHeader: {
+    color: "white",
+    textTransform: "uppercase",
+  },
+}));
+
+const LetsTalk: React.FC<Props> = ({}) => {
+  const classes = useStyles();
+
+  return (
+    <Box pt={16} height="100vh">
```

Comment:
We might as well add this padding to the `View` component so we don't have to do this for each view.